src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -37,6 +37,7 @@ class Yaml
     public const DUMP_NUMERIC_KEY_AS_STRING = 4096;
     public const DUMP_NULL_AS_EMPTY = 8192;
     public const DUMP_COMPACT_NESTED_MAPPING = 16384;
+    public const DUMP_FORCE_QUOTES = 32768;
```

Comment:
```suggestion
    public const DUMP_FORCE_DOUBLE_QUOTES = 32768;
```

Maybe like this?

Reply:
I would have say `DUMP_FORCE_DOUBLE_QUOTES_ON_VALUES`? Per [this comment](https://github.com/symfony/symfony/pull/59880#pullrequestreview-2652333314)

Reply:
Ah indeed, makes sense